src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operator) => {
+  const first = Big(numberOne);
+  const second = Big(numberTwo);
+
+  let result = 0;
+
+  switch (operator) {
+    case '+':
+      result = first.plus(second);
+      break;
+    case '-':
+      result = first.minus(second);
+      break;
+    case '×':
+      result = first.times(second);
+      break;
+    case '÷':
+      result = second !== 0 ? first.div(second) : 0;
+      break;
```

Comment:
return `INFINITY` or `NaN` or `ZERO DIVISION ERROR`

something like that

Reply:
Replaced